plugins/lando-core/lib/utils.js

```diff
@@ -147,3 +148,42 @@ exports.stripWild = versions => _(versions)
   .map(version => (version.split('.')[2] === 'x') ? _.slice(version.split('.'), 0, 2).join('.') : version)
   .value();
 
+
+exports.formattedOptions = {
+  output: {
+    describe: 'Output in given format: json',
+    alias: ['o'],
+    string: true,
+  },
+  path: {
```

Comment:
@pirog I think if we were fetching a single piece of data from a flat data structure, `field` would make good sense. But since we could be fetching complex data from far within a complex data structure, I think `path` better communicates what you're getting and what you're transversing. It also takes inspiration from the `path` argument name used with [_.get()](https://lodash.com/docs/4.17.11#get) that this command is a thin wrapper around.

Reply:
@pianomansam totally fine with using `path`, but if we are going to do that i think we should consider removing `field` and/or making it a hidden alias for backwards compatibility.

Reply:
That actually clears up my concern on using `format` instead of `output` 😄

Reply:
@pianomansam perfect so sounds like next steps are just

1. Remove `field` in favor of `path`, optionally spin up a "hidden" alias for `path` called `field` for backwards compat
2. Change `output` to `format`

Let's dial those in and then i can help you get rolling on the testing stuff! Thanks a bunch @pianomansam, always great to have contributors who know WTF they are doing ;)

Reply:
@pirog Another snag: The `list` command already uses `-f` as an alias for `--filter` 😢

Reply:
bleh maybe we can just alias `--field` and forget `-f` for now. This is a pretty new and probably not oft used feature so we can likely get away with not having full backwards compat

Reply:
@pirog So you're suggesting add the new `format` option but don't include an alias for it? This way we leave both `field` and `filter` in place?

Reply:
oh whoops sorry, i thought you were talking about the hidden implicit aliases for `path` so that it would be backwards compat with the `field` we are removing. 

Easiest thing would be to just not give `format` an alias which i think is fine.

Reply:
@pirog The latest commit renames `output` to `format` and removes the alias.